### PR TITLE
Move mocks for gtest_paralell_tests.py to another file.

### PR DESCRIPTION
**Pull request by ehlemur-zz**
What do you think?
I think it might make things easier to read.

### CODE REVIEW

File: gtest_parallel_mocks.py [hunk 1]
```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python2
```
Comment (pbos) [inline gtest_parallel_mocks.py]: This doesn't need to be executable so you can remove this line, right? (probably applies for all but `gtest-parallel`)

File: gtest_parallel_mocks.py [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    self.test_binary = test_id[0]
+    self.test_name = test_id[1]
+    self.task_id = (test_id[0], test_id[1], execution_number)
+
+  def run(self):
+    pass
+
+
+@contextlib.contextmanager
+def guard_temp_dir():
```
Comment (pbos) [inline gtest_parallel_mocks.py]: This is not a mock. Either move this and below to another file (back to _tests) or rename this file something like `gtest_parallel_test_utils.py`. WDYT?

**Review by pbos — COMMENTED** [review]
Looks good overall.

File: gtest_parallel_mocks.py (hunk 2 above)
Comment (pbos) [reply gtest_parallel_mocks.py]: I think I prefer moving these and below back to the top of the test file (gtest_parallel_mocks.py is a good name). Up to you though.

**Comment by ehlemur-zz** [comment]
PTAL :)

**Pull request merged**